### ISSUE TITLE
testing

### ISSUE THREAD

**Issue by madiharris** [resilienseas/resilienseas.github.io#1]
Hey @bbest,

what's up with https://github.com/resilienseas/resilienseas.github.io/blob/master/scripts/gapanalysis.R#L32-L34?

**Comment by bbest** [comment]
Hey @madiharris & @raetaylorburns, 

Nice work today! So I can comment, but looks like I cannot close the issue because of permissions. The easiest would be to make us all Owners per https://help.github.com/articles/permission-levels-for-an-organization/. For now only @madiharris  has this level.

**Comment by madiharris** [comment]
Hey @bbest,

Ahhhh - I see that now! I gave you and @raetaylorburns owner status. Let me know if that change didn't work

**Comment by raetaylorburns** [comment]
Hi @bbest!
Thanks for all your GitHub help yesterday! We rasterized the monitoring inventory polygons and set up the framework for calculating variability, but we ran into an issue when trying to assign a monitoring site measurement to the rasterized polygons in this step https://github.com/resilienseas/resilienseas.github.io/blob/fe5248f441eb437eacd568bc5138a342a541d7df/scripts/gapanalysis.R#L155-L156. We think it is because the length of the SST dataframe is different from the number of values in the raster. Any ideas on how to address this issue?
Thanks!
